CONTRIBUTING.md

```diff
@@ -87,69 +87,158 @@ You may need to install the dev_requirements for this
+
+`azext_iot_central_primarykey` is the IoT Central Application Id or name. Optional variable, used for IoT Central tests. Cannot be currently retrieved through the CLI.
+
+`azext_iot_central_storage_container` is the name of blob container belonging to the `azext_iot_teststorageaccount` storage account. Optional environment variable, defaults to 'central' when not specified.
+
+##### Digital Twin Test variables
+
+`azext_dt_region` is the region to restrict Digital Twins creation. Optional variable, if not specified, will create the Digital Twins in a supported region.
+
+`azext_dt_adx_cluster` is the name of the Azure Data Explorer Cluster to use. Optional variable, used for Digital Twin data history tests. Azure Data Explorer Cluster creation time can take up to 20 minutes, so having a cluster is recommended.
+
+`azext_dt_adx_database` is the name of the database in the Azure Data Explorer Cluster to use. Optional variable, used for Digital Twin data history tests.
+
+`azext_dt_adx_rg` is the resource group that contains the Azure Data Explorer Cluster. If not provided, `az_iot_testrg` will be used as the resource group.
+
+`azext_dt_ep_eventgrid_topic` is the Event Grid Topic to use. Optional variable, used for Digital Twin endpoint tests.
+
+`azext_dt_ep_servicebus_namespace` is the Service Bus Namespace to use. Optional variable, used for Digital Twin endpoint tests.
+
+`azext_dt_ep_servicebus_policy` is the policy for the topic in the Service Bus Namespace to use. Optional variable, used for Digital Twin endpoint tests.
+
+`azext_dt_ep_servicebus_topic` is the topic in the Service Bus Namespace to use. Optional variable, used for Digital Twin endpoint tests.
+
+`azext_dt_ep_eventhub_namespace` is the Event Hub Namespace to use. Optional variable, used for Digital Twin endpoint and data history tests.
+
+`azext_dt_ep_eventhub_policy` is the policy for the Event Hub instance in the Event Hub Namespace to use. Optional variable, used for Digital Twin endpoint and data history tests.
+
+`azext_dt_ep_eventhub_topic` is the Event Hub instance in the Event Hub Namespace to use. Optional variable, used for Digital Twin endpoint and data history tests.
+
+`azext_dt_ep_eventhub_topic_consumer_group` is the Service Bus Namespace to use. Optional variable, used for Digital Twin data history tests.
+
+`azext_dt_ep_rg` is the resource group that contains the endpoint (Event Hub, Event Grid, Service Bus) variables. Optional variable, if not provided, `az_iot_testrg` will be used as the resource group.
+
 ##### IoT Hub
 
 Execute the following command to run the IoT Hub integration tests:
 
-`pytest azext_iot/tests/iothub/ -k "_int"`
+`pytest azext_iot/tests/iothub/ -k "_int.py"`
 
 ##### Device Provisioning Service
 
 Execute the following command to run the IoT Hub DPS integration tests:
 
-`pytest azext_iot/tests/dps/ -k "_int"`
+`pytest azext_iot/tests/dps/ -k "_int.py"`
```

Comment:
Thoughts about combining the 'executing the service tests' sections with the 'setting the service specific env vars to run tests' sections?

Reply:
I reformatted how the tests are presented. Please look over the changes